ci/src/main/scala/org/typelevel/sbt/TypelevelCiPlugin.scala

```diff
@@ -126,6 +126,39 @@ object TypelevelCiPlugin extends AutoPlugin {
     githubWorkflowJavaVersions := Seq(JavaSpec.temurin("8"))
   )
 
+  override def projectSettings: Seq[Setting[_]] = Seq(
+    Test / Keys.executeTests := {
+      val results: Tests.Output = (Test / Keys.executeTests).value
+      GitHubActionsPlugin.appendtoStepSummary(renderTestResults(Keys.name.value, results))
+      results
+    }
+  )
+
+  def renderTestResults(projectName: String, results: Tests.Output): String = {
+    val tableHeader: String =
+      s"### $projectName Tests Result\n" +
```

Comment:
🚲 🏠 : `Tests Result` or `Tests Results` or `Suite Results` ?

Reply:
They all seem fine. One other idea is to give the command that can be used to run that locally e.g.
`++scalaVersion projectName/test`

Reply:
Good Idea